### DISCUSSION TITLE
After not building my Ghidra repo for a week or two and sync'ing in new changes, I can't get it to build anymore.

### DISCUSSION THREAD

**Discussion by hippietrail** [NationalSecurityAgency/ghidra#7268] [Q&A]
I've been working on Loaders and FileSystems in GhidraDev on an official release for a while and just tried sync'ing up my local Ghidra repo to build here.

But now I'm getting many build errors, a place I've been before but was always able to deal with [following these steps](https://github.com/NationalSecurityAgency/ghidra/discussions/7050#discussioncomment-10950774)

1. Select all in the Package Explorer, right click, and then click delete. Don't check the checkbox on the dialog.
2. Run gradle cleanEclipse eclipse
3. File -> Import -> General -> Existing Projects into Workspace
4. Root should be the top of your git repo
5. Make sure only the 1st and 4th Options are checked
6. Finish

I've gone through these steps twice and the result is the same: 
<img width="1051" alt="image" src="https://github.com/user-attachments/assets/777b4893-9191-49cb-b03c-fdb173895379">

Am I doing something wrong? I'm looking through the [Developer Guide](https://github.com/NationalSecurityAgency/ghidra/blob/master/DevGuide.md) but nothing is jumping out at me.

**Comment by ryanmkurtz** [comment]
There errors are related to building the GhidraDev Eclipse plugin.  If you are not modifying that code, i recommend you just delete those two Eclipse projects.

**Reply by hippietrail** [reply]
Thanks. I had to delete several of the debugger projects too.

**Reply by ryanmkurtz** [reply]
Yes, deleting some debugger projects is expected.
